routes/handler.go

```diff
@@ -0,0 +1,19 @@
+package routes
+
+import (
+	"github.com/gorilla/mux"
+
+	"go-rest-api/controllers"
+)
+
+const VERSION = "/api/v1"
+
+//handler requests
```

Comment:
The format to comment exported functions is: `<functionName> <description of what it does and/or returns>` for example for this one: `HandlerRequests - Returns router that handles API paths`. I know, it can get redundant at times